### ISSUE TITLE
Unnecessary '1 results' spoken after pressing CTRL+g with screen reader enabled

### ISSUE THREAD

**Issue by lukaszgo1** [microsoft/vscode#175559]
Type: <b>Bug</b>  cc @isidorn 
1. Start NVDA and VS Code - ensure that accessibility support is enabled
2. Open an empty file, press CTRL_g, and listen to the screen reader output

Expected: Screen reader announces the name of the newly focused control
Actual: Before announcing the new focus there is an unnecessary announcement provided via live region which says 'one results'. Aside from the fact that it should say 'one result' this is distracting and in that particular context seems to serve no purpose.

VS Code version: Code - Insiders 1.76.0-insider (ead95b4a7014d12ddcb2dd6ddcb16e73f1e4cd67, 2023-02-24T21:55:13.530Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-5200U CPU @ 2.20GHz (4 x 2195)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.91GB (11.55GB free)|
|Process Argv|--crash-reporter-id fd55c502-07ef-4f56-8594-165b200772c3|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
isort|ms-|2023.9.10461023
python|ms-|2023.3.10581011
vscode-pylance|ms-|2023.2.43
jupyter|ms-|2023.2.1000561009
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.15
vscode-jupyter-cell-tags|ms-|0.1.6
vscode-jupyter-slideshow|ms-|0.1.5

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30422396
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
vscod805:30301674
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30404738
cppdebug:30492333
vsclangdf:30492506
c4g48928:30535728
dsvsc012:30540252
pynewext54:30618038
pylantcb52:30590116
pyindex848:30611229
nodejswelcome1:30587009
pyind779:30611226
pythonsymbol12:30651887
6233i204:30659907
pythonb192:30661256
pythonms35:30671666

```

</details>

**Comment by lukaszgo1** [comment]
I do not understand why this was marked as a duplicate of #166920. This issue reports an unnecessary announcement done via live region which can be easily confirmed by disabling dynamic content changes in NVDA. There is no information's missing, just the announcement about item being selected is distracting and adds no value.
